mungegithub/mungers/matchers/comment/finder.go

```diff
@@ -16,11 +16,25 @@ limitations under the License.
 
 package comment
 
-import "github.com/google/go-github/github"
+import (
+	"time"
+
+	"github.com/google/go-github/github"
+)
 
 // FilteredComments is a list of comments
 type FilteredComments []*github.IssueComment
 
+// GetLast returns the last comment in a series of comments
+func (f FilteredComments) GetLast() *github.IssueComment {
```

Comment:
No I disagree, you have Empty() to check if it's empty. I'd rather this function have a strong semantic